package.json

```diff
@@ -9,8 +9,11 @@
 		"webpack": "webpack --mode=production"
 	},
 	"license": "MIT",
-	"dependencies": {},
+	"dependencies": {
+		"sanitize-html": "^2.7.0"
```

Comment:
Should probably be a peer dependency?

Reply:
Yes, the documentation says, you have to install your own dependencies.